### PR TITLE
Tiny edits to contributing/setup

### PR DESCRIPTION
**Pull request by jpoon**
Was going through the `setup.md` and noticed a couple of typos

In addition, a couple of questions:

* `brew install protobuf` installs 3.1.0. Not sure if that version is sufficient? 
* I would liek to add more around how to dev/test. I'm new to go, so how do you guys generally develop: edit files -> `make build` -> `go test` -> manually test newly built `pachctl`?

### CODE REVIEW

File: contributing/setup.md [hunk 1]
```diff
@@ -17,7 +17,7 @@ If you're contributing to pachyderm, you may need the following additional setup
 
 ## General Requirements
 
-Follow the [general setup instructions](https://github.com/pachyderm/pachyderm/blob/master/SETUP) to install:
+Install:
```
Comment (jpoon) [inline contributing/setup.md]: Previous link to https://github.com/pachyderm/pachyderm/blob/master/SETUP is dead. Couldn't find a similar looking "SETUP" file so deleting it all together

File: contributing/setup.md [hunk 2]
```diff
@@ -43,12 +43,6 @@ And you'll stay up to date!
 
 ## Protoc
```
Comment (jpoon) [inline contributing/setup.md]: proto-compilation looked to be fixed and merged. This comment may no longer be relevant?

**Comment by jdoliner** [comment]
All looks good, many thanks for the pull @jpoon.
Last step is I need you to sign our [CLA](https://pachyderm.wufoo.com/forms/pachyderm-contributor-license-agreement/) and I can get this merged in.

**Comment by jpoon** [comment]
@jdoliner Done.

**Pull request merged**